src/kohlrahbi/new_maus/edifact_components.py

```diff
@@ -144,33 +119,30 @@ class ValuePoolEntry(BaseModel):
     - (key: "332", meaning: "DE, DVGW", ahb_expression: "X")
     """
 
-    #: the qualifier in edifact, might be e.g. "E01", "D", "9", "1.1a", "G_0057"
-    qualifier: str = Field(..., description="The qualifier in EDIFACT")
-    meaning: str = Field(
+    qualifier: Annotated[
+        str,
+        StringConstraints(strip_whitespace=True, min_length=1),
+    ] = Field(..., description="The qualifier in EDIFACT")
+    meaning: Annotated[str, StringConstraints(strip_whitespace=True, min_length=1)] = Field(
         ...,
         description="The meaning as it is written in the AHB,"
         "e.g. 'Einzug', 'Entwurfs-Version', 'GS1', 'Codeliste Gas G_0057'",
     )
-    ahb_expression: str = Field(
+    ahb_expression: Annotated[str, StringConstraints(strip_whitespace=True, min_length=1)] = Field(
         ...,
         description="The AHB expression, in most cases this is a simple 'X'; it must not be empty",
     )
 
     @field_validator("qualifier")
     @classmethod
-    def validate_qualifier(cls, v):
+    def validate_qualifier(cls, v: Any) -> str:
```

Comment:
```suggestion
    def validate_qualifier(cls, v: str) -> str:
```
Warum so lose typisiert?

Reply:
good point, war müde und wollte nur mypy happy machen und bin dem Beispiel in der Doku gefolgt.
Fixt in acaf980e86235d57bafa10527d7486b08411653e